infra/app/service/main.tf

```diff
@@ -22,12 +22,20 @@ data "aws_subnets" "private" {
 }
 
 locals {
+  # The prefix key/value pair is used for Terraform Workspaces, which is useful for projects with multiple infrastructure developers.
+  # By default, Terraform creates a workspace named “default.” If a non-default workspace is not created this prefix will equal “default”,
+  # if you choose not to use workspaces set this value to "dev"
```

Comment:
I don't understand this comment. What does it mean to "choose not to use workspaces"?

Reply:
Oh funny. I copy/pasted this directly from https://github.com/navapbc/template-infra/blob/7a9b157a16d380bddcffb23692061b1caa9f5fb4/infra/app/app-config/env-config/main.tf#L2-L5

Reply:
what was i thinking

Reply:
looks great